### PR TITLE
LBSD-1350 Bug when trying to type the URL manually

### PR DESCRIPTION
**Pull request by m0g**
> In authoring click to add an embed.
> Observe: as soon as the first character is placed in the embed field it is set and further typing is not possible.

My approach to fix that is to trigger the change function on paste action or when pressing enter/return. Therefore when somebody is typing something in the input field, the embed retrieval routine will only be triggered when pressing the return key

### CODE REVIEW

File: client/app/scripts/ng-sir-trevor-blocks.js [hunk 1]
```diff
@@ -141,6 +141,7 @@ define([
                         input = fixSecureEmbed(input);
                         // if the input is an url, use embed services
                         if (isURI(input)) {
+                            console.log('is uri');
```
Comment (nistormihai) [inline client/app/scripts/ng-sir-trevor-blocks.js]: not needed

File: client/app/scripts/ng-sir-trevor-blocks.js [hunk 2]
```diff
@@ -156,7 +157,15 @@ define([
                         } else {
                             that.loadData({html: input});
                         }
-                    }, 200));
+                    }
+                    this.$editor.on('paste', _.debounce(callServiceAndLoadData, 200));
+
+                    this.$editor.on('keydown', function(e) {
+                        if (e.keyCode == 13) {
```
Comment (nistormihai) [inline client/app/scripts/ng-sir-trevor-blocks.js]: should fail on jshint

**Comment by m0g** [comment]
> but just to be sure we can have some sort of information regarding this ( placeholder or just an info text above when starting to type)

@nistormihai you mean to indicate the user that he needs to press `enter` to load the embed?

**Comment by djbrd-sourcefabric** [comment]
:+1:

**Pull request merged**